src/test/index.ts

```diff
@@ -2,7 +2,32 @@ import * as path from 'path';
 import * as Mocha from 'mocha';
 import * as glob from 'glob';
 
+function setupNyc() {
+	const NYC = require('nyc');
+	const defaultExclude = require('@istanbuljs/schema/default-exclude');
+	const cwd = path.join(__dirname, '..', '..', '..');
```

Comment:
[F] This is probably incorrect

Reply:
I added a console log locally. i tis giving me  `/home/apupier/git/camel-lsp-client-vscode` which is the root folder of the whole extension.